src/aws/ec2-wrapper.ts

```diff
@@ -80,22 +64,22 @@ export class EC2Wrapper {
     );
 
     if (subnets.length === 0) {
-      throw new Error("Invalid subnet name, it does not exist");
+      throw new Error(`Subnets with vpc id '${vpcId}', tag key '${tagKey}' and tag values '${tagValues}' do not exist`);
     }
 
-    const missingSubnetNames = subnetNames.filter((subnetName) => {
+    const missingSubnetValues = tagValues.filter((tagValue) => {
       // collect subnets by name
       const subnetsByName = subnets.filter((subnet) => {
-        const nameTag = subnet.Tags.find((tag) => tag.Key === "Name");
-        return wildcardMatches(subnetName, nameTag.Value);
+        const nameTag = subnet.Tags.find((tag) => tag.Key === tagKey);
+        return wildcardMatches(tagValue, nameTag.Value);
       });
       return subnetsByName.length === 0;
     });
 
-    if (missingSubnetNames.length) {
-      throw new Error(
-        `Subnets do not exist for the names: ${missingSubnetNames}. ` +
-        "Please check the names are correct or remove it."
+    if (missingSubnetValues.length) {
+      Globals.logError(
```

Comment:
Why change this to log an error instead of throwing one?

Reply:
Also is it possible to combine this with the error at https://github.com/amplify-education/serverless-vpc-discovery/pull/43/files#diff-a13ce4e50f6a01157d391a478768f3ff4813e433346a3d9f31684819ac4a0629R67

Maybe
 ```typescript
if (missingSubnetValues.length || subnets.length === 0) {
```

Reply:
Agree